pkg/util/blacklist.go

```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"errors"
+	"strings"
+)
+
+var blacklistedCommands = []string{
```

Comment:
Shall we keep it as map bool, so that checks would be faster?

Reply:
Yes, that would be faster, let me apply the changes